sdks/python/apache_beam/transforms/environments.py

```diff
@@ -240,15 +240,16 @@ def __init__(
     super(DockerEnvironment, self).__init__(capabilities, artifacts)
     if container_image:
       logging.info(
-        'Using provided Python SDK container image: %s' % (container_image))
+          'Using provided Python SDK container image: %s' % (container_image))
       self.container_image = container_image
     else:
-      logging.info('No image given - inferring default Python SDK container image')
+      logging.info(
+          'No image given - inferring default Python SDK container image')
```

Comment:
This would be a good place to tell the user how to provide an image if they want to.

Reply:
hmm I'm not sure it's a good idea to link the possible env/runner flags to the log message here, but at least changed the wording a bit

Reply:
I prefer not to say `container_image` since it could be misconstrued as a flag. Best to leave it how you had it originally.